rancid-tomatillos/src/App.js

```diff
@@ -3,27 +3,43 @@ import React, {Component} from "react";
 import movieData from "./sample-data/movie-data.js"
 import Navbar from "./Components/Navbar/Navbar"
 import CardContainer from "./Components/CardContainer/CardContainer"
+import MovieView from "./Components/MovieView/MovieView"
 
 class App extends Component {
   constructor() {
     super();
     this.state= {
       allMovies: movieData.movies,
+      isClicked: false,
+      selectedMovie: {}
+      }
     };
-  }
+  
+
+   handleClick = (id) => {
+     let selectedMovie = this.state.allMovies.find(movie => movie.id === id)
+     this.setState({selectedMovie: selectedMovie, isClicked: true})
+      console.log("SELECTED MOVIE", selectedMovie)
+      console.log("ID", id);
+    }
 
+  returnToMain = () => {
+    console.log("RTM firing")
+    this.setState({isClicked:false, selectedMovie: {}})
+  }
```

Comment:
Nice work clearing state to return to the main page.  This may change a bit now that you have React Router!